grants_policy.md

```diff
@@ -0,0 +1,94 @@
+Policy:
+
+All applications for funding of the CWL project (grant applications, and
+similar) must be submitted by Software Freedom Conservancy (SFC) unless
+an agreement otherwise has been approved by both the CWL Project
+Leadership Team and SFC itself.
```

Comment:
> However this does not stop anyone from contributing to any aspect of the CWL project. It just means they can't claim to represent the project in financial settings (without permission).

But that's not how it is currently written.  It doesn't say "all applications claiming to represent the project" it says "all applications for funding of the CWL project".  So should it be rewriten this way?

> All applications for funding claiming to represent the CWL project (grant applications, and
similar) must be submitted by Software Freedom Conservancy (SFC) unless
an agreement otherwise has been approved by both the CWL Project
Leadership Team and SFC itself.

Reply:
I feel that these concerns have been satisfied in the latest draft.